gr-uhd/apps/uhd_app.py

```diff
@@ -295,5 +298,7 @@ def cslist(string):
                             help="Show asynchronous message notifications from UHD")
         group.add_argument("--sync", choices=('default', 'pps', 'auto'),
                           default='auto', help="Set to 'pps' to sync devices to PPS")
+        group.add_argument("--clock-source",choices=('internal', 'external', 'gpsdo', 'default'),
```

Comment:
Simply remove the choices here. However, you can add them in the help string ("Set the clock source (e.g., internal, external, gpsdo, ...)").